### DISCUSSION TITLE
Not using prebuilts but storage keeps increasing with email warnings

### DISCUSSION THREAD

**Discussion by ne0rrmatrix** [community/community#77739] [Codespaces]
Bug

I most likely have something misconfigured or I am doing something wrong. Or it is possible this is a bug. But I have been getting monthly warning about using up storage for prebuilts. As far as I know I am not using prebuilts. I checked my account and I am using over 11gb's or storage and I have no idea what is using that. How can I prevent the usage and/or what is causing this. It has been going on for months and months.

**Comment by erickdc7** [comment]
It sounds like you're experiencing an issue where your GitHub storage usage is increasing despite not intentionally using prebuilts. Here's what you can do to address this:

- **Check for Artifacts**: Review your repositories for any GitHub Actions workflow runs that may have generated artifacts. These can consume storage space even if they're not visible in your account.
- **Review Repository Settings**: Ensure that your repository settings do not automatically generate prebuilts or other large files that could be taking up space.
- **Contact GitHub Support**: If you're unable to identify the cause of the storage increase, reach out to GitHub Support for assistance. They can help you investigate and resolve the issue.
- **Monitor Storage Usage**: Regularly monitor your storage usage in the Billing and plans section of your GitHub settings to keep track of any changes and manage your data effectively.

Remember to check all repositories associated with your account, as the storage usage could be spread across multiple projects.
